src/components/Gradebook/index.jsx

```diff
@@ -406,32 +436,70 @@ export default class Gradebook extends React.Component {
-                      <span className="label">
-                        Assignment Types:
-                      </span>
-                      <InputSelect
-                        name="assignment-types"
-                        ariaLabel="Assignment Types"
-                        value={this.props.selectedAssignmentType}
-                        options={this.mapAssignmentTypeEntries(this.props.assignmentTypes)}
-                        onChange={this.updateAssignmentTypes}
-                        disabled={this.props.assignmentFilterOptions.length === 0}
-                      />
-                    </div>
-                    <div className="student-filters">
-                      <span className="label">
-                        Assignment:
-                      </span>
-                      <InputSelect
-                        name="assignment"
-                        ariaLabel="Assignment"
-                        value={this.props.selectedAssignment}
-                        options={this.getAssignmentFilterOptions()}
-                        onChange={this.handleAssignmentFilterChange}
-                        disabled={this.props.assignmentFilterOptions.length === 0}
-                      />
-                    </div>
+                    <Collapsible title="Assignments" isOpen>
+                      <div>
+                        <div className="student-filters">
+                          <span className="label">
+                            Assignment Types:
+                          </span>
+                          <InputSelect
+                            name="assignment-types"
+                            ariaLabel="Assignment Types"
+                            value={this.props.selectedAssignmentType}
+                            options={this.mapAssignmentTypeEntries(this.props.assignmentTypes)}
+                            onChange={this.updateAssignmentTypes}
+                            disabled={this.props.assignmentFilterOptions.length === 0}
+                          />
+                        </div>
+                        <div className="student-filters">
+                          <span className="label">
+                            Assignment:
+                          </span>
+                          <InputSelect
+                            name="assignment"
+                            ariaLabel="Assignment"
+                            value={this.props.selectedAssignment}
+                            options={this.getAssignmentFilterOptions()}
+                            onChange={this.handleAssignmentFilterChange}
+                            disabled={this.props.assignmentFilterOptions.length === 0}
+                          />
+                        </div>
+                        <p>Grade Range (0% - 100%)</p>
+                        <form className="d-flex justify-content-between" onSubmit={this.handleSubmitAssignmentGrade}>
+                          <InputText
```

Comment:
Olga asked me to include a % sign next to each input - something like this. Could you include that with these inputs too? I imagine she'd want that here as well.

![image](https://user-images.githubusercontent.com/11871801/62909352-3b33f080-bd4a-11e9-94c5-c4da35d672b5.png)

Reply:
I can, but if the goal is consistency b/t them it may be better for you to do it, since I can't see your code as of right now.